clients/admin-ui/cypress/e2e/datasets.cy.ts

```diff
@@ -13,14 +13,13 @@ describe("Dataset", () => {
   });
 
   describe("List of datasets view", () => {
-    // TODO: Update Cypress test to reflect the nav bar 2.0
-    it.skip("Can navigate to the datasets list view", () => {
+    it("Can navigate to the datasets list view", () => {
       cy.visit("/");
-      cy.getByTestId("nav-link-Datasets").click();
+      cy.contains("nav a", "Data map").click();
+      cy.contains("nav a", "Manage datasets").click();
```

Comment:
My _personal_ preference is to use `contains` like this, when it's reasonable. I think adding testid's where needed is totally fine too, I just didn't want to expand the scope to modifying the `NavTopBar` component to include those as part of this and felt that navigating based on the (legible) text is pretty reasonable here. All the other navigation tests behave similarly.